external/scala/test.sh

```diff
@@ -17,7 +17,7 @@ source $(dirname "$0")/test_base_functions.sh
 echo_setup
 
 TEST_SUITE=$1
-export JAVA_TOOL_OPTIONS="-Dfile.encoding=UTF8"
+export JAVA_TOOL_OPTIONS="$JAVA_TOOL_OPTIONS -Dfile.encoding=UTF8"
```

Comment:
Same as jenkins.sh, no need to do the prefix.

Reply:
not true, as stated above, or I'm missing something. java_tool_options is widely used JDK's variable. Overwitign is impolite (and buggy)